tests/test_npy_cache.py

```diff
@@ -0,0 +1,112 @@
+from pathlib import Path
+import yaml
+
+from cloudpathlib import S3Path
+
+from zamba.data.video import (
+    VideoLoaderConfig,
+    npy_cache,
+    get_cached_array_path,
+    load_video_frames,
+)
+
+config_yaml = """
+    cache_dir: local_data/cache
+    crop_bottom_pixels: 50
+    early_bias: false
+    ensure_total_frames: true
+    evenly_sample_total_frames: false
+    fps: 4.0
+    frame_indices: null
+    frame_selection_height: null
+    frame_selection_width: null
+    i_frames: false
+    megadetector_lite_config:
+        confidence: 0.25
+        fill_mode: score_sorted
+        image_height: 640
+        image_width: 640
+        n_frames: 16
+        nms_threshold: 0.45
+        seed: 55
+        sort_by_time: true
+    model_input_height: 240
+    model_input_width: 426
+    pix_fmt: rgb24
+    scene_threshold: null
+    total_frames: 16
+    cleanup_cache: false
+    cache_dir: data/cache
+"""
+
+
+def test_get_cached_array_path():
+    config_dict = yaml.safe_load(config_yaml)
+    config = VideoLoaderConfig(**config_dict)
+
+    # NOTE: the validation in VideoLoaderConfig changes some fields, so this fails
```

Comment:
yes this is a design issue with some of the configs, which change fields rather than setting properties based on the inputted fields. let's remove this commented out assert

Reply:
Half joking: that's not a commented out line of code, it's part of a note.

I agree that it's often good to remove commented out lines of code, but it is also often useful to include code snippets in documentation and notes. If I want to say, "the following line of code demonstrates the point I am making", I suppose I could put the line of code in quotes to make it clear that it is not a forbidden commented out line of code.